packages/sdk/src/types.ts

```diff
@@ -1187,18 +1234,18 @@ export enum AccountTransactionType {
     UpdateBakerStake = 6,
     UpdateBakerRestakeEarnings = 7,
     UpdateBakerKeys = 8,
-    UpdateCredentialKeys = 13,
-    EncryptedAmountTransfer = 16,
-    TransferToEncrypted = 17,
-    TransferToPublic = 18,
-    TransferWithSchedule = 19,
-    UpdateCredentials = 20,
-    RegisterData = 21,
-    TransferWithMemo = 22,
-    EncryptedAmountTransferWithMemo = 23,
-    TransferWithScheduleAndMemo = 24,
-    ConfigureBaker = 25,
-    ConfigureDelegation = 26,
+    UpdateCredentialKeys = 9,
+    EncryptedAmountTransfer = 10,
+    TransferToEncrypted = 11,
+    TransferToPublic = 12,
+    TransferWithSchedule = 13,
+    UpdateCredentials = 14,
+    RegisterData = 15,
+    TransferWithMemo = 16,
+    EncryptedAmountTransferWithMemo = 17,
+    TransferWithScheduleAndMemo = 18,
+    ConfigureBaker = 19,
+    ConfigureDelegation = 20,
```

Comment:
These changes make the unit tests fail, as one test tests the expected transaction hash of a transaction. To be honest I have a hard time understanding how the previous values were accepted by the node? The changed values correspond to the values in the rust SDK' and in the GRPC api proto types.

Reply:
According to the doc comment, these values are the ones used for the tag in the serialization, so you should not look at the gRPC types.
The values should match [the tag found here](https://github.com/Concordium/concordium-base/blob/main/rust-src/concordium_base/src/transactions.rs#L1037)

Reply:
Huh.. I'll revert this then 😅